src/biogeochem/CNPhenologyMod.F90

```diff
@@ -2760,8 +3046,13 @@ subroutine CNOffsetLitterfall (num_soilp, filter_soilp, &
                      repr_grainn_to_seed(p,k) = t1 * cropseedn_deficit_to_restore
 
                      ! Send the remaining grain to the food product pool
+                     repr_grainc_to_food_thispool = cpool_to_reproductivec(p,k) - repr_grainc_to_seed(p,k)
                      repr_grainc_to_food(p,k) = t1 * reproductivec(p,k) &
-                          + cpool_to_reproductivec(p,k) - repr_grainc_to_seed(p,k)
+                          + repr_grainc_to_food_thispool
```

Comment:
Just a note that this refactoring is another possible source of answer changes - so if you can't determine the cause for all of the answer changes, you could try temporarily going back to having this expression for repr_grainc_to_food_thispool inlined and seeing if that removes some answer changes.